prepare.py

```diff
@@ -25,8 +25,9 @@ def main():
 
     env_builder = venv.EnvBuilder(with_pip=True)
     venv_context = env_builder.ensure_directories(venv_path)
+    requirements_path = _ROOT_DIR / 'dev_requirements.txt'
 
-    python_run(venv_context, "pip", "install -r {}".format(_ROOT_DIR / 'dev_requirements.txt'))
+    python_run(venv_context, "pip", ["install", "-r", "{}".format(_ROOT_DIR / 'dev_requirements.txt')])
```

Comment:
Do you actually need to do a format here?(same in install.py)

Reply:
i can also use. a str cast. need to sinceit's a Path object, not a string. Might as well changeitto astr cast forreadibility